cfgov/jinja2/v1/_includes/organisms/ask-search.html

```diff
@@ -18,20 +18,23 @@
 
    placeholder: String; text to enter for the search input's placeholder value.
 
+   label_level:  String; heading level to use for the label.
+
    ========================================================================== #}
 
 {% macro render(
     ask_query='',
     language='en',
     show_label=True,
     autocomplete=True,
-    placeholder=''
+    placeholder='',
+    label_level='h3'
```

Comment:
Instead of allowing an arbitrary number of heading levels into this component, let's be more atomic and limit the number of variations this component allows. Let's make this a boolean, `is_compact`, or similar and then where you set the heading levels you'd check the boolean:

```
{% if is_compact %}
<h3>{{ _('Search for your question') }}</h3>
{% else %}
<h2>{{ _('Search for your question') }}</h2>
{% endif %}
```
Also, maybe just check it around the tags so you don't repeat the heading text twice, but I'll leave that up to you.

Reply:
@niqjohnson Ans and I had a conversation around those lines before he posted his comment. Where we landed was that the thing probably needs a significant overhaul, and moving of some of the parts of it into fields on the Wagtail block (e.g., if we had a `HeadingBlock` in there, this template logic would be unnecessary), but given the "urgency" of this request, I would not hold up a short-term solution for future proofing.

The idea behind Ans's specific suggestion was to attempt to limit the possible proliferation of more variations on the component in the interim between now and doing that overhaul.

Reply:
@anselmbradford, @Scotchester: I fixed this in fb44e25. I changed the string `label_level` to the boolean `is_subsection`. Hopefully `is_subsection` as the name for that property captures the intent: is the form rendered inside an existing section of a page that already has its own `h2` or should it be a top-level section on the page that needs an `h2`. I can change the label if you can think of a better name.

Also, I used a ternary to handle the template logic, but I can change that to a regular if/else block if that seems confusing.

Reply:
LGTM 👍 Nice job!